AdminX "About" modal

peterzimon opened the pull request:
refs. https://github.com/TryGhost/Product/issues/3949

peterzimon committed:
Added static about modal

peterzimon committed:
Added static about modal

peterzimon committed:
Added link to version number

peterzimon committed:
Added link to version number

ronaldlangeveld committed:
Wired up about config data

no issue

ronaldlangeveld committed:
Wired up about config data

no issue

ronaldlangeveld committed:
Cleaned up comments

no issue

ronaldlangeveld committed:
Cleaned up comments

no issue

ronaldlangeveld committed:
Added Upgrade Status to About modal

refs https://ghost.slack.com/archives/C0568LN2CGJ/p1696922777143079

- This wires up the UpgradeStatus from Ember to the React Application to
  tell Self Hosters to upgrade in the about page.
- There's a lot of logic around this currently handled by the Ember
  application, in which for now it's better in reuse that.

ronaldlangeveld commented:
@peterzimon I have another [PR open](https://github.com/TryGhost/Ghost/pull/18555) for the "update available" logic + data. We can merge this one in first to avoid conflicts.

ronaldlangeveld committed:
Merge branch 'added-notifications-x' into admin-x-about

peterzimon committed:
Formatted 'Update available' msg in About modal

ronaldlangeveld committed:
Added `dangerouslySetInnerHTML` to upgrade note

no issue

- this is required to get the HTML coming from the API to render
  correctly in React.
